docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,112 @@
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can select which repositories can be accessed by the GitHub app.
+
+Creating a .dockstore.yml File
+-------------------------------
+
+Once the GitHub app is installed on the correct repo, the next step is to create a ``/.dockstore.yml`` file. We'll cover a very straigtforward example
+first, but depending on how you configured the workflow during registration and whether your GitHub repository houses multiple workflows published on Dockstore,
+there will be additional steps to writing your ``/.dockstore.yml`` file.
+
+Let's say we have the following CWL workflow registered on Dockstore that came from this `repository <https://github.com/NatalieEO/ghapps-single-workflow>`__ and you would like to convert the master branch.
+
+.. figure:: /assets/images/docs/single-workflow-to-migrate.png
+   :alt: Workflow to Migrate
+
+As noted in our other documentation, create a ``/.dockstore.yml`` file in the root directory of the master branch in your repository. The file should look like the following
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test/dockstore.cwl.json
+
+The information above was filled out using the following:
+
+- ``subclass`` is taken from the ``Descriptor Type``
+- ``primaryDescriptorPath`` is from ``Workflow Path``
+- ``testParameterFiles`` is from ``Test File Path``
+
+During the original registration for your workflow, you may have filled out the ``Workflow Name`` field shown in the picture below.
+
+.. figure:: /assets/images/docs/workflow-name-field.png
+   :alt: Workflow to Migrate
+   :width: 60 %
+
+Typically this field is used when you want to register multiple workflows from the same repo, but you may have filled it out regardless. To check if the workflow
+you want to migrate has a workflow name set, select the workflow and look at the title on top as shown in the picture below.
+
+.. image:: /assets/images/docs/workflow-title.png
+
+The title consists of:
+``<sourceControl>/<organization name>/<repository name>/<optional workflow name>:<version name>``
+
+If you see a workflow name inserted, you must include the name field in your ``/.dockstore.yml``.
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test/dockstore.cwl.json
+        name: optional-name
+
+If you have multiple workflows stemming from the same repo, a single ``/.dockstore.yml`` can be used to convert them. You should also keep the information about
+the Workflow Name in mind because all but one workflow must have this field set (e.g. if you have three workflows from the same repo registered on Dockstore, at least two of them will have the Workflow Name set).
```

Comment:
I think its more than keep in mind, right? Doesn't the migration fail in the multiple registered workflow migration if the all the names don't match up? I think we need to be explicit that it will fail so people will know that this is what they need to debug.